src/playbin.js

```diff
@@ -1,33 +1,71 @@
 /* exported Playbin */
 
-const {Gdk, GLib, GObject, Gtk, Gst} = imports.gi;
+const {GLib, GObject, Gtk, Gst} = imports.gi;
+
+const Lock = GObject.registerClass({
+    CssName: 'lock',
+}, class Lock extends Gtk.Button {
+    _init(props) {
+        super._init(props);
+
+        this.canDefault = false;
+        this.canFocus = false;
+
+        /* Make button invisible */
+        this.connect('draw', cr => {
+            const style = this.get_style_context();
+            const c = this.get_clip();
+            Gtk.render_background(style, cr, c.x, c.y, c.width, c.height);
```

Comment:
How does this make the button invisible like the comment says? Also, I understand that code is here because the theme sets a background on the button, but buttons always draw a background in GTK, so you should not need the override...